### PR TITLE
MM-35125 CRT: Mention badge on threads doesn't appear until refresh

### PR DESCRIPTION
**Pull request by reflog**
#### Summary

post notifications for thread followers, not 'participants'

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-35125
#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by zefhemel — APPROVED** [review]
LGTM from a code perspective.

File: store/sqlstore/thread_store.go [hunk 1]
```diff
@@ -307,6 +307,21 @@ func (s *SqlThreadStore) GetThreadsForUser(userId, teamId string, opts model.Get
...
+		Select("ThreadMemberships.UserId").
+		From("ThreadMemberships").
+		Where(sq.Eq{"PostId": threadID}).ToSql()
+	_, err := s.GetReplica().Select(&users, query, args...)
+
+	if err != nil {
+		return nil, err
+	}
+	return users, nil
+}
```
Comment (ashishbhate) [inline store/sqlstore/thread_store.go:323]:
```suggestion
	return s.GetReplica().Select(&users, query, args...)
}
```

**Review by ashishbhate — APPROVED** [review]
LGTM. Sorry about the noise - I deleted the wrong suggestion.

**Review by ogi-m — APPROVED** [review]
LGTM

**Pull request merged**